src/decorators/withConfig/validate.js

```diff
@@ -33,7 +33,9 @@ export default function validate(config = {}) {
    *
    * Resulting in: `nl-BE-VWV`, `nl-BE` or `nl`
    */
-  if (!/^[a-z]{2}(?:-[A-Z]{2}(?:[- ]{1}[A-Z09]{1,3})?)?$/.test(config.locale)) {
+  if (
```

Comment:
Everything works as expected, but could you update the information comment above this line? Either remove the stuff that isn't required anymore or maybe a better solution is to add a note that we added the i flag as a temporary fix for older safari browsers?